packages/fx-core/resource/package.nls.json

```diff
@@ -819,7 +818,7 @@
   "depChecker.useGlobalDotnet": "Using dotnet from PATH:",
   "depChecker.dotnetInstallStderr": "dotnet-install command failed without error exit code but with non-empty standard error.",
   "depChecker.dotnetInstallErrorCode": "dotnet-install command failed.",
-  "depChecker.NodeNotFound": "Cannot find Node.js.\n\nTeams Toolkit requires Node.js; the recommended version is @NodeVersion.\n\nClick \"Learn more\" to learn how to install the Node.js.\n\n(If you just installed Node.js, restart Visual Studio Code for the change to take effect.)",
+  "depChecker.NodeNotFound": "Cannot find Node.js. The supported node versions are specified in the package.json. Go to https://nodejs.org/about/releases/ to install a supported Node.js. Restart all your Visual Studio Code instances after the installation is finished.",
```

Comment:
It's not recommended to include link in strings, better use a placeholder.

Reply:
Thx! I will updated all the nodejs link in another PR.